src/python/cli_new/lib/cli/mesos.py

```diff
@@ -199,11 +199,14 @@ def __init__(self, master, task_id):
                         "This command is only supported for tasks"
                         " launched by the Universal Container Runtime (UCR).")
 
+        # Get the scheme of the agent
```

Comment:
Might be more readable:
```
scheme = "https://" if config.agent_ssl() else "http://"
```

Reply:
Oh yes, that's pretty. Didn't know that. :-) Thanks @cf-natali